src/Client/config/amplitude_api_key.yml

```diff
@@ -0,0 +1 @@
+amplitudeApiKey: ec6a87cc68232d908b4398fad6793139
```

Comment:
Is it ok to have this publicly released?

Reply:
This needs to be in the codebase as this will be run from the console, if we had an option to run it from within Drupal, we could have saved it in the config probably. Maybe we can obfuscate the API Key but I'm not sure if that can be done in this scenario.

Reply:
I am not familiar with Amplitude but I think the same as @jose-oliveira, that this should not be released publicly. I am just brainstorming here but would it be an option to get this key from Plexus (through an endpoint)? At least there would be an authentication on the request and only people could get it who has valid ACH credentials.

Reply:
Yeah, we could think on a way to hide it... some other projects at Acquia throw it straight to the code though, like this https://github.com/acquia/cli/blob/408603bf22397667b0df1cdb35a2defa6d5c9cc9/src/Helpers/TelemetryHelper.php#L80